### PR TITLE
Made `_mvt` REST handler public in serverless

### PR DESCRIPTION
**Pull request by craigtaverner**
Apparently all REST API's need an explicit @ServerlessScope annotation in order to be public in serverless contexts. I labeled this `enhancement` because I found similar PRs for other search API's what added this annotation as an enhancement.

### CODE REVIEW

**Comment by iverase** [comment]
I would prefer to nark this as a non-issue (contrary to your observation, the ones I saw are marked as non-issue). Otherwise LGTM . I wait for @martijnvg for reviewing it.

File: docs/changelog/97405.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+pr: 97405
```
Comment (martijnvg) [inline docs/changelog/97405.yaml:1]: When labelling this as a non issue, this changelog can be removed.

**Review by martijnvg — APPROVED** [review]
LGTM

File: docs/changelog/97405.yaml (hunk 1 above)
Comment (craigtaverner) [reply docs/changelog/97405.yaml]: When I changed the label, the changelog was removed, and a rebuild did not happen - cool!

Comment (craigtaverner) [reply docs/changelog/97405.yaml]: Spoke too soon. Rebuild running. I'll wait for that to finish before merging.

**Pull request merged**